tmuxp/__about__.py

```diff
@@ -1,6 +1,6 @@
 __title__ = 'tmuxp'
 __package_name__ = 'tmuxp'
-__version__ = '1.5.7post0'
+__version__ = '1.5.7post1'
```

Comment:
@joseph-flinn Can you make this 1.5.8?

```suggestion
__version__ = '1.5.8'
```

Reply:
yep